drivers/SmartThings/matter-lock/src/new-matter-lock/init.lua

```diff
@@ -1549,7 +1507,6 @@ local new_matter_lock_handler = {
       [capabilities.lock.commands.unlock.NAME] = handle_unlock,
     },
     [capabilities.lockUsers.ID] = {
-      [capabilities.lockUsers.commands.addUser.NAME] = handle_add_user,
```

Comment:
Why was this removed again? `addUser` is a command in the capability so I think we should have a command handler for it

Reply:
In fact, the `adduser` command is not used when creating a new user. All use `addCredential` to add a new user. And there is no `userindex` parameter in the `adduser` command, so driver needs to find and create an empty user index. this process is complicated.
If you think I have to maintain the function, please let me know.

Reply:
Hi Hunsup, out of curiosity do you know where `addCredential` is being leveraged?

Reply:
Please ignore this- I was checked out on the wrong branch. Apologies for the confusion.

Reply:
@HunsupJung I do think we need a handler for all the commands in the capability since other things besides the app can call capability commands and since `addUser` is a command on this capability it should be implemented.

Reply:
I updated addUser command and some sequence.